product_docs/docs/postgres_distributed_for_kubernetes/1/installation_upgrade.mdx

```diff
@@ -72,3 +88,5 @@ creating PGD clusters. See the
 If you're trying to install EDB Postgres Distributed for Kubernetes on Red Hat OpenShift,
 see [Red Hat OpenShift](openshift.md), which contains
 information on the certified operator maintained by EDB.
+import { startActivity } from "gatsby-cli/lib/reporter/redux/actions"
```

Comment:
I don't think you want this here do you? Looks like a fragment or accidental autocomplete.

Reply:
<img src="https://media0.giphy.com/media/3ohzdYJK1wAdPWVk88/giphy.gif"/>